core/src/main/kotlin/plugins/OpenApi.kt

```diff
@@ -113,29 +116,53 @@ fun Application.configureOpenApi() {
             generator = { type ->
                 type
                     .collectSubTypes()
-                    .processReflection {
+                    .analyzeTypeUsingReflection {
                         // Replace Instants with Strings in the generated schema to avoid breaking changes in the UI.
                         // This might later be replaced with a proper schema for dates.
-                        redirect<Instant, String>()
-                        redirect<Instant?, String?>()
+                        typeRedirects += TypeRedirect(
```

Comment:
You should be able to replace this with

    redirect {
        from<Instant>()
        to<String>()
    }

Similar below.

Reply:
Thanks, that's better.